src/main.cpp

```diff
@@ -507,13 +507,14 @@ int main(int argc, char *argv[])
         { "log",             no_argument,       NULL, 'l' },
         { "audio",           optional_argument, NULL, 'a' },
         { "to-yuv",          no_argument,       NULL, 't' },
+        { "help",            no_argument,       NULL, 'h' },
         { 0,                 0,                 NULL,  0  }
     };
 
     int c, i;
     std::string param;
     size_t pos;
-    while((c = getopt_long(argc, argv, "o:f:g:c:p:d:la::t::", opts, &i)) != -1)
+    while((c = getopt_long(argc, argv, "o:f:g:c:p:d:la::t::h::", opts, &i)) != -1)
```

Comment:
From the getopt_long man page `Two colons mean an option takes an optional arg`. This option doesn't take an optional argument, does it?

Reply:
Sure misclick on that. Btw can I change the options to be in alphabetical order ?